addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -295,13 +295,21 @@ odoo.define('website.s_website_form', function (require) {
             // force server date format usage for existing fields
             this.$target.find('.s_website_form_field:not(.s_website_form_custom)')
             .find('.s_website_form_date, .s_website_form_datetime').each(function () {
+                const $input = this.querySelector('input');
```

Comment:
When your variable holds an element, it should be suffixed by `El`.
Prefixed variable by `$` is for jQuery element.